packages/ui/input/checkbox/index.jsx

```diff
@@ -10,11 +12,13 @@ import styles from "./index.css";
 const variants = {
   checkbox: styles.checkbox,
   switch: styles.switch,
+  indeterminate: styles.indeterminate,
```

Comment:
@ailincaibianca `indeterminate` shouldn't be a `variant`
It should be a `boolean` property of a `checkbox` variant, same as `checked`

`<Checkbox variant="checkbox" indeterminate={true/false} />`